frozendict/__init__.py

```diff
@@ -1,6 +1,7 @@
 import collections
-import operator
-import functools
+
+
+iteritems = getattr(dict, 'iteritems', dict.items)
```

Comment:
If it's being left at module-scope, perhaps it should be renamed to `_iteritems` ? (to make it clear it's not part of the public API)